graspy/simulations/sims_sample_corr.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from graspy.simulations import sample_edges
+import copy
+import warnings
+
+def sample_corr(P, Rho, directed=False, loops=False):
+    """
+    Generate a correlated Erdos-Renyi graph G2 based on G1 graph
+    with Bernoulli distribution.
+
+    Both G1 and G2 are binary matrices. 
+    If the value of one position in graph 1 matrix is 1, 
+    the probability of the same position of graph 2 to be 1 
+    follows the Bernoulli distribution of
+    (P[i][j])+Rho[i][j]*(1-P[i][j]).
```

Comment:
not necessary to go into the implementation of how to make the graphs correlated